packages/widget-reader/src/ReaderTextPart.vue

```diff
@@ -32,16 +36,25 @@
 
 <script>
   import URN, { Icon } from '@scaife-viewer/common';
-  import { MODULE_NS, SELECT_LINE } from '@scaife-viewer/store';
+  import {
+    MODULE_NS,
+    SELECT_LINE,
+    SELECT_SCHOLION,
+  } from '@scaife-viewer/store';
   import ReaderToken from './ReaderToken.vue';
 
   export default {
     props: ['textPart'],
     components: { Icon, ReaderToken },
     methods: {
-      onLineSelect() {
+      async onLineSelect() {
+        // reset scholion in case one was selected in the ScholiaWidget
+        await this.$store.dispatch(`${MODULE_NS}/${SELECT_SCHOLION}`, {
+          scholion: null,
+        });
+
         this.$store.dispatch(`${MODULE_NS}/${SELECT_LINE}`, {
-          ref: `${this.urn.version}${this.textPart.ref}`,
+          ref: this.textPart.ref,
```

Comment:
This change is ok for now, but if we were working with the `SELECT_LINE` action in a context where say, we were showing two different versions, I'd prefer a "fully-qualified" reference (e.g., the text part URN).

Reply:
Ah, right -- this was a bit shortsighted on my part. I was trying to avoid having to parse the URN in multiple places, but your approach makes more sense. I can definitely fix this in a subsequent pass.